pkg/api/customization/kontainerdriver/actionhandler.go

```diff
@@ -70,3 +83,51 @@ func (a ActionHandler) setDriverActiveStatus(apiContext *types.APIContext, statu
 
 	return err
 }
+
+func (lh ListHandler) LinkHandler(apiContext *types.APIContext, next types.RequestHandler) error {
+	k8sCurr := strings.Split(settings.KubernetesVersionsCurrent.Get(), ",")
+	rkeSysImages, winSysImages := map[string]v3.RKESystemImages{}, map[string]v3.WindowsSystemImages{}
+	for _, k8sVersion := range k8sCurr {
+		rkeSysImg, err := kd.GetRKESystemImages(k8sVersion, lh.SysImageLister, lh.SysImages)
+		if err != nil {
+			return err
+		}
+		rkeSysImages[k8sVersion] = rkeSysImg
+		// ui allows only >= 1.14 for windows
+		k8sMajor := util.GetTagMajorVersion(k8sVersion)
+		if k8sMajor <= "v1.13" {
+			continue
+		}
+		winSysImg, err := kd.GetRKEWindowsSystemImages(k8sVersion, lh.WinSysImageLister, lh.WinSysImages)
```

Comment:
if we have UI allowing only >= 1.14 for windows, why do we generate the System Images for earlier versions?

Reply:
we don't, they have been removed from the metadata windows file

Reply:
So do we need to have a conditional check here for 1.13 and below? Instead should kd.GetRKEWindowsSystemImages() return nil or some error indicating images unavailable? In future if we deprecate the images for some later versions, we wont have to change this code if we start returning empty/unavailable response.

Reply:
`kd.GetRKEWindowsSystemImages()` would return error in this case, but we have no way of knowing if this is because there is some actual error in fetching the image (in which case we want to return this to user) or it's just something we should not be fetching in the first place. 

There's only one `k8s-current` for both General and Windows and UI bypasses it by hardcoding it... so we need to sadly hardcode too?